SpatialOS/launch_configs/deployment.json

```diff
@@ -1,5 +1,5 @@
 {
-  "template": "small",
+  "template": "w2_r0500_e5",
```

Comment:
Could you put this in a separate PR instead as it has nothing to do with 14.0 and should (normally) have been done a while ago. 🙂

Reply:
Reverted the template change.

Reply:
Ok, I fucked up. I accidentally merged the PR thinking that this was reverted. I even removed the changelog entry. Sorry about that.

Reply:
@JonasImprobable if you could please create another PR with the changelog that'll be alright. 🙂